sphinx/registry.py

```diff
@@ -21,7 +21,11 @@
 from docutils.parsers import Parser
 from docutils.parsers.rst import Directive
 from docutils.transforms import Transform
-from pkg_resources import iter_entry_points
+
+try:  # Python < 3.10 (backport)
+    from importlib_metadata import entry_points
+except ImportError:
+    from importlib.metadata import entry_points
```

Comment:
Is there a reason the builtin isn't tried before the backport?

Reply:
Yes: the code is using APIs that were changed in 3.10, so for 3.8 and 3.9 we want to use the backport over the standard library module. :slightly_smiling_face:

Reply:
Got it. Thanks for the context! 😄